Classes/PBCommitList.m

```diff
@@ -43,7 +43,10 @@ - (void)keyDown:(NSEvent *)event
 	}
 	else if ([character rangeOfCharacterFromSet:[NSCharacterSet characterSetWithCharactersInString:@"jkcv"]].location == 0)
 		[webController sendKey: character];
-	else
+	else if (([character characterAtIndex:0] == NSDownArrowFunctionKey)
+			 && [event modifierFlags] & NSControlKeyMask) {
+		[controller selectParentCommit:self];
+	} else
```

Comment:
I think adding a menu item is the right option for new keyboard shortcuts in basically every case, but I can see that not adding one matches the existing behavior of table view arrow shortcuts not having menu items. Text editing navigation shortcuts also don't have menu items.

Capturing ⌥-↑ as a no-op might also be a good idea, so that you don't easily lose your place by mistyping it. There's also Control- as a prefix for the arrows, which I think doesn't do anything on tables by default.